packages/client/hmi-client/src/components/workflow/scenario-templates/policy-design/tera-policy-design-template.vue

```diff
@@ -0,0 +1,151 @@
+				placeholder="Select an intervention for this model"
+				@update:model-value="props.scenario.setInterventionPolicyId($event, modelConfiguration)"
+				class="mb-3"
+				:loading="isFetchingModelData"
+			/>
+
+			<label>Select a dataset (optional)</label>
+			<Dropdown
+				:model-value="props.scenario.getDatasetId()"
+				:options="allDatasetOptions"
+				option-label="assetName"
+				option-value="assetId"
+				placeholder="Select a dataset (optional)"
+				@update:model-value="props.scenario.setDatasetId($event)"
+				class="mb-3"
+			/>
+		</template>
+		<template #outputs>
+			<label :class="{ 'disabled-label': _.isEmpty(modelStateAndObsOptions) || isFetchingModelData }"
+				>Select an output metric</label
+			>
+			<MultiSelect
+				:disabled="_.isEmpty(modelStateAndObsOptions) || isFetchingModelData"
+				:model-value="selectedOutputSettings"
+				placeholder="Select output metrics"
+				:options="modelStateAndObsOptions"
+				@update:model-value="scenario.setOptimizeOutputSettings($event)"
+				:loading="isFetchingModelData"
+				filter
+			/>
+			<img :src="horizon" alt="Horizon scanning chart" class="" />
+		</template>
+	</tera-scenario-template>
+</template>
+
+<script setup lang="ts">
+import Dropdown from 'primevue/dropdown';
+import { computed, ref, watch } from 'vue';
+import { useProjects } from '@/composables/project';
+import { AssetType, ModelConfiguration, InterventionPolicy, Model } from '@/types/Types';
+import { getInterventionPoliciesForModel, getModel, getModelConfigurationsForModel } from '@/services/model';
+import _ from 'lodash';
+import { getModelConfigurationById } from '@/services/model-configurations';
+import horizon from '@/assets/svg/template-images/horizon-thumbnail.svg';
+import MultiSelect from 'primevue/multiselect';
+import TeraScenarioTemplate from '../tera-scenario-template.vue';
+import { ScenarioHeader } from '../base-scenario';
+import { PolicyDesignScenario } from './policy-design-scenario';
+
+const props = defineProps<{
+	scenario: PolicyDesignScenario;
+}>();
+
+const header: ScenarioHeader = Object.freeze({
+	title: 'Policy design template',
+	question: 'Search for a policy/intervention that achieves a desired outcome',
+	description:
+		'Requires a desired future outcome state and a variable to intervene on. The system will attempt to optimize the variable to achieve the desired outcome state and provide an intervention for that variable (if possible). If there is data to calibrate against, the suggested intervention will be more accurate. This is a type of counterfactual analysis.',
+	examples: [
+		'What percentage vaccination is required to have hospitalizations below 5% of the population in three months?',
+		'How much would gas taxes need to raise to reduce driving by 5% by October?',
+		'How much would property taxes need to lower to increase investment by %5 in one year?'
+	]
+});
+
+const allModelOptions = computed(() => useProjects().getActiveProjectAssets(AssetType.Model));
```

Comment:
For add'l IDE type hinting / checking we can type these computeds, e.g.

```suggestion
const allModelOptions = computed<ProjectAsset[]>(() => useProjects().getActiveProjectAssets(AssetType.Model));
```